Dockerfile

```
@@ -0,0 +1,32 @@
+FROM alpine:3.12
+
+COPY ./wlc/ /wlc
+COPY ./completion/ /completion
+COPY ./setup.cfg /setup.cfg
+COPY ./setup.py /setup.py
+COPY ./wl /wl
+COPY ./LICENSE /LICENSE
+COPY ./requirements.txt /requirements.txt
```

This IMHO creates 7 additional Docker layers, what doesn't sound good. Also some of the files are not really useful in the image (LICENSE or wl). I also don't think completion is useful inside the image...

```suggestion
COPY ./wlc/ setup.cfg setup.py requirements.txt /app
```